app/views/projects/_form.html.erb

```diff
@@ -52,66 +43,89 @@
       </div>
       <%= error_span(@project[:public]) %>
     </div>
-    <table class='table table-striped'>
-      <thead>
-      <tr>
-        <th><%= t('projects.form.wimi', default: 'Research Assistants:') %></th>
-        <th><%= t '.actions', default: t('helpers.actions') %></th>
-      </tr>
-      </thead>
-      <tbody>
-        <% @project.users.order('last_name').each do |user| %>
-          <% if user.is_wimi?%>
-            <tr>
-              <td>
-                <%= user.name %>
-              </td>
-              <td>
-                <% if can? :edit, @project%>
-                  <% if user == current_user %>
-                    <%= link_to t('projects.form.sign_out_of_project', default:'Sign out of project'), sign_user_out_project_path(user_id: user.id, id: @project), method: :delete, class: 'btn btn-danger btn-xs', id: 'SignOutMyself' %>
-                  <% else %>
-                    <%= link_to t('projects.form.sign_out_of_project', default:'Sign out of project'), sign_user_out_project_path(user_id: user.id, id: @project), method: :delete, class: 'btn btn-danger btn-xs', id: 'SignOut' %>
-                  <% end %>
-                <% end %>
-              </td>
-            </tr>
-          <% end %>
-        <% end %>
-      </tbody>
-    </table>
 
-    <table class='table table-striped'>
-      <thead>
-      <tr>
-        <th><%= t('projects.form.hiwi', default:'Student Assistants:') %></th>
-        <th><%= t '.actions', default: t('helpers.actions') %></th>
-      </tr>
-      </thead>
-      <tbody>
-        <% @project.users.order('last_name').each do |user| %>
-          <% if user.is_hiwi?%>
-            <tr>
-              <td>
-                <%= user.name %>
-              </td>
-              <td>
-                <% if can? :edit, @project%>
-                  <% if user == current_user %>
-                    <%= link_to t('projects.form.sign_out_of_project', default:'Sign out of project'), sign_user_out_project_path(user_id: user.id, id: @project), method: :delete, class: 'btn btn-danger btn-xs', id: 'SignOutMyself' %>
-                  <% else %>
-                    <%= link_to t('projects.form.sign_out_of_project', default:'Sign out of project'), sign_user_out_project_path(user_id: user.id, id: @project), method: :delete, class: 'btn btn-danger btn-xs', id: 'SignOut' %>
-                  <% end %>
-                <% end %>
-              </td>
-            </tr>
+    <% if @project.persisted? %>
+      <div class="form-group">
+        <table class='table table-striped' width="1000px">
+          <thead>
+          <tr>
+            <th width="600px"><%= t('projects.form.wimi', default: 'Research Assistants:') %></th>
```

Comment:
Ich wüsste nicht, wann Rails mal in die Lage kommt zum Default-Wert zurückgreifen zu müssen. Also IMO können die alle weg.